.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,40 +13,37 @@ Read and tick the following checkbox after you have created the issue or place a
 
 * [ ] I have read the [CONTRIBUTING.md](https://github.com/nextcloud/news/blob/master/CONTRIBUTING.md) and followed the provided tips
 * [ ] I accept that the issue will be closed without comment if I do not check [here](https://github.com/nextcloud/news/blob/master/CONTRIBUTING.md#before-reporting-bugs)
-* [ ] I accept that the issue will be closed without comment if I don not fill out all items in the issue template.
+* [ ] I accept that the issue will be closed without comment if I do not fill out all items in the issue template.
 
 
 ## Explain the Problem
 What problem did you encounter?
 
 ## Steps to Reproduce
 Explain what you did to encounter the issue
-1.
-2.
-3.
+1. Click
+2. Drag
+3. ??
+4. Profit!
```

Comment:
Sure that we want to prefill it with values?
Maybe we could use `Explain step by step what you did to encounter the issue` instead as a description.

Reply:
In the preview the old template showed as numbers at the end of the line so I figured I'd add values to fix that